src/AutomatedCar/SystemComponents/Packets/HMIPacket.cs

```diff
@@ -150,5 +151,43 @@ public void AccSet(bool on)
                 this.acc = !this.acc;
             }
         }
+
+        public void GasPedalIncrease(double tick, int limit = 100)
+        {
+            Increase(tick, ref this.gaspedal, limit);
+        }
+
+        public void GasPedalDecrease(double tick, int limit = 100)
+        {
+            Decrease(tick, ref this.gaspedal, limit);
+        }
+
+        public void BrakePedalIncrease(double tick, int limit = 100)
+        {
+            Increase(tick, ref this.breakpedal, limit);
+        }
+
+        public void BrakePedalDecrease(double tick, int limit = 100)
+        {
+            Decrease(tick, ref this.breakpedal, limit);
+        }
+
+        private void Increase(double tick, ref double pedal, int limit = 100)
+        {
+            do
+            {
+                Thread.Sleep(Convert.ToInt32(1000 / tick));
+                pedal++;
```

Comment:
Ezt nem egészen értem, vár a program 1/60 másodpercet és utána növeli a double értéket egyel ? 
azért lett a pedál és a kormány double hogy lehessen törtekkel növelni pl 1/60 al.

Reply:
ez igaz, javitom is mindjart